clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/TitusClient.java

```diff
@@ -66,6 +67,12 @@
   public void updateDisruptionBudget(
       JobDisruptionBudgetUpdateRequest jobDisruptionBudgetUpdateRequest);
 
+  /**
+   * @param serviceJobProcessesRequest
+   * @return
```

Comment:
no `@return` since it's `void`.

This code already has a bunch of empty comments... my opinion is if the javadoc (or another comment) doesn't add value it shouldn't be there.

Reply:
👍